### ISSUE TITLE
Error rates of matconvnet-alex are higher than what have been reported. 

### ISSUE THREAD

**Issue by jstudy** [vlfeat/matconvnet#615]
The pre-trained model of matconvnet-alex is said to reach 41.8% top-1 err. and 19.2% top- 5 err. on the ILSVRC 2012 validation data, with  one single centre-crop evaluation. However, my validation errors are much higher than the above results. I used  the given alexnet of beta20, with  'simplenn' model, 'batchNormalization',  'gaussian' weightInitMethod , and  'cudnn '. And I run for 20, 30, 40, epochs in logspace learning rate , respectively.  However, the validation errors are almost the same, about 44.8%  top-1 err. and 21.8%  top-5 err.  Can anyone figure this out?

**Comment by lenck** [comment]
What cuDNN version are you using? There may be subtle differences in them...

**Comment by jstudy** [comment]
Hi, I am using cuDNN5.  The reported results used cuDNN4. There should not have such a big difference.   Maybe  the values of parameters are different. Would you like to present the detailed settings of the parameters.
